tests/integration/api/mutations/checkout/anonymousCheckout.test.js

```diff
@@ -0,0 +1,501 @@
+import encodeOpaqueId from "@reactioncommerce/api-utils/encodeOpaqueId.js";
+import importAsString from "@reactioncommerce/api-utils/importAsString.js";
+import TestApp from "/tests/util/TestApp.js";
+
+const AvailablePaymentMethodsQuery = importAsString("./AvailablePaymentMethodsQuery.graphql");
+const AddCartItemsMutation = importAsString("./AddCartItemsMutation.graphql");
+const AnonymousCartByCartIdQuery = importAsString("./AnonymousCartByCartIdQuery.graphql");
+const RemoveCartItemsMutation = importAsString("./RemoveCartItemsMutation.graphql");
+const CreateCartMutation = importAsString("./CreateCartMutation.graphql");
+const UpdateFulfillmentOptionsForGroupMutation = importAsString("./UpdateFulfillmentOptionsForGroupMutation.graphql");
+const SelectFulfillmentOptionForGroupMutation = importAsString("./SelectFulfillmentOptionForGroupMutation.graphql");
+const SetShippingAddressOnCartMutation = importAsString("./SetShippingAddressOnCartMutation.graphql");
+const UpdateCartItemsQuantityMutation = importAsString("./UpdateCartItemsQuantityMutation.graphql");
+const PlaceOrderMutation = importAsString("./PlaceOrderMutation.graphql");
+const PublishProductToCatalogMutation = importAsString("./PublishProductsToCatalogMutation.graphql");
+const SetEmailOnAnonymousCart = importAsString("./SetEmailOnAnonymousCartMutation.graphql");
+
+
+jest.setTimeout(300000);
+
+const encodeProductOpaqueId = encodeOpaqueId("reaction/product");
+
+const internalShopId = "123";
+const internalProductId = "999";
+const opaqueProductId = "cmVhY3Rpb24vcHJvZHVjdDo5OTk="; // reaction/product:999
```

Comment:
why not use `const opaqueProductId = encodeOpaqueId("reaction/product", 999);`?

Reply:
Didn't encode since this style was used for pretty much all of the other tests. If this is something we want to change, all those other tests should be on the radar for an update at some point.

Reply:
we have both styles, for all the tests I have written I use the encoding utility. Not a hill to die on, but I believe it makes it clear how the value was computed if we use the util function.

Reply:
Updated to use the util